From khali@linux-fr.org Sat Nov 26 19:24:41 2005
Date: Sat, 26 Nov 2005 20:48:35 +0100
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Cc: Laurent Riffard <laurent.riffard@free.fr>, Takashi Iwai <tiwai@suse.de>
Subject: [PATCH 16/25] i2c: Drop i2c_driver.{owner,name}, 7 of 11
Message-Id: <20051126204835.38fb0268.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-drop-driver-owner-and-name-07-ppc.patch

From: Laurent Riffard <laurent.riffard@free.fr>

We should use the i2c_driver.driver's .name and .owner fields
instead of the i2c_driver's ones.

This patch updates the drivers for ppc arch.

Signed-off-by: Laurent Riffard <laurent.riffard@free.fr>
Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 sound/ppc/keywest.c |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

--- gregkh-2.6.orig/sound/ppc/keywest.c
+++ gregkh-2.6/sound/ppc/keywest.c
@@ -41,7 +41,10 @@ static int keywest_attach_adapter(struct
 static int keywest_detach_client(struct i2c_client *client);
 
 struct i2c_driver keywest_driver = {  
-	.name = "PMac Keywest Audio",
+	.driver = {
+		.owner = THIS_MODULE,
+		.name = "PMac Keywest Audio",
+	},
 	.id = I2C_DRIVERID_KEYWEST,
 	.attach_adapter = &keywest_attach_adapter,
 	.detach_client = &keywest_detach_client,
